dist/travisci/.travis.yml

```diff
@@ -24,8 +24,12 @@ env:
 
 before_install:
   - echo 'sendmail_path = /bin/true' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - sudo echo "deb http://archive.ubuntu.com/ubuntu focal main restricted universe multiverse" >> /etc/apt/sources.list
+  - sudo apt-get update
+  - sudo apt-get -y install sqlite3/focal
```

Comment:
sqlite version for php 7.4 is very old and Drupal 9 requires a newer one.

Reply:
Changed the approach to run things inside the docker container so no longer needed.